broadlink/__init__.py

```diff
@@ -699,10 +699,44 @@ def __init__(self, *args, **kwargs):
     # Returns decrypted payload
     # New behaviour: raises a ValueError if the device response indicates an error or CRC check fails
     # The function prepends length (2 bytes) and appends CRC
+
+    def calculate_crc16(input_data):
```

Comment:
I think this method requires a @staticmethod annotation (or include self as the first argument)